Modules-Index.md

```diff
@@ -0,0 +1,91 @@
+## Hi there 👋
+
+### Documentation: [LLazyEmail Documentation](https://llazyemail.github.io/documentation/)
+
+<!--
+
+**Here are some ideas to get you started:**
+
+🙋‍♀️ A short introduction - what is your organization all about?
+🌈 Contribution guidelines - how can the community get involved?
+👩‍💻 Useful resources - where can the community find your docs? Is there anything else the community should know?
+🍿 Fun facts - what does your team eat for breakfast?
+
+-->
+
+Email Generator Template Universe
+
+## Main project
+
+[Markdown-to-email](https://github.com/LLazyEmail/markdown-to-email)
+
+Modules for Markdown-to-Email project
+
+```
+"atherdon-callbacks": "^1.10.0",
+"atherdon-newsletter-html-replacer": "^0.4.0",
+"atherdon-newsletter-js-layouts": "^1.5.1",
+"atherdon-newsletter-react": "^1.0.11",
+"atherdon-newsletter-react-replacer": "^0.4.0",
+"atherdon-old-newsletter-js-outertemplate": "^2.4.0",
+"markup-generator": "^0.4.0",
+```
+
+### Generator Module
+
+[Generator-markdown-to-html](https://github.com/LLazyEmail/generator-markdown-to-html)
+
+## Markdown-regex module
+
+[Markdown-regex](https://github.com/LLazyEmail/markdown-regex)
+
+### Plain JS Template aka `atherdon-old-newsletter-js-outertemplate`
+
+[Trying-lit](https://github.com/LLazyEmail/_trying-lit)
+
+### `React DOM` Template 
+
+[React-plain-template-example](https://github.com/LLazyEmail/react-plain-template-example)
+
+### Factory object for Display Classes
+
+[Template-object](https://github.com/LLazyEmail/template-object)
+
+### Example repository for cloning
+
+[Default-email-template](https://github.com/LLazyEmail/default-email-template)
+
+- [Generator-module](https://github.com/LLazyEmail/generator-module)
+
+- [Replacer](https://github.com/LLazyEmail/replacer)
+
+- [Pixel-email-cli](https://github.com/LLazyEmail/pixel-email-cli)
+
+### React One Template
+
+[React-email-template](https://github.com/LLazyEmail/react-email-template)
+
+![mindmap](https://github.com/atherdon/markdown-to-email/blob/main/MindMap1.png?raw=true)
+
+### React Design System
+
+- [React-email-typography-components](https://github.com/LLazyEmail/react-email-typography-components)
+- [Nextjs-preview](https://github.com/LLazyEmail/nextjs-preview)
+- [React-email-miscellaneous-components](https://github.com/LLazyEmail/react-email-miscellaneous-components)
+- [React-sendgrid-form](https://github.com/LLazyEmail/react-sendgrid-form)
+- [React-email-template](https://github.com/LLazyEmail/react-email-template)
+- [React-email-table](https://github.com/LLazyEmail/react-email-table)
+- [Email-template-react](https://github.com/LLazyEmail/email-template-react)
+- [React-email-letter-components](https://github.com/LLazyEmail/react-email-letter-components)
+
+### Cool projects to see
```

Comment:
We can remove this section

Reply:
https://github.com/LLazyEmail/documentation/pull/163